bin/run-fisher.sh

```diff
@@ -14,4 +16,5 @@ python -u DeepSpeech.py \
   --epoch=20 \
   --display_step 1 \
   --validation_step 5 \
+  --checkpoint_dir "$XDG_DATA_HOME/$(basename ${0%.*})" \
```

Comment:
This contains the `run-` prefix which I think is not desirable. Ideally you could do this in one step
```bash
...
--checkpoint_dir "$XDG_DATA_HOME/$(basename -s .sh ${0#run-})" \
...
```
But that doesn't seem to work due to some bash foo I don't have.

So one can do it in two steps.
```bash
XDG_DATA_HOME=...
CHECKPOINT_DIR=$(basename ${0%.*})
CHECKPOINT_DIR=${CHECKPOINT_DIR#run-}
...
   --validation_step 5 \
   --checkpoint_dir "$XDG_DATA_HOME/$CHECKPOINT_DIR" \
...
```